LGE Nuxus 5X (Android 7.1.1, API 25)

Monitorのエラーは以下。

<details>
03-17 21:15:23.690 28679-29824/? E/cr_tabmodel: State file does not exist.
03-17 21:15:24.335 29935-29935/io.github.akameco.qiitax E/JavaBinder: !!! FAILED BINDER TRANSACTION !!!  (parcel size = 881128)
03-17 21:15:24.340 29935-29935/io.github.akameco.qiitax E/AndroidRuntime: FATAL EXCEPTION: main
                                                                          Process: io.github.akameco.qiitax, PID: 29935
                                                                          java.lang.RuntimeException: android.os.TransactionTooLargeException: data parcel size 881128 bytes
                                                                              at android.app.ActivityThread$StopInfo.run(ActivityThread.java:3781)
                                                                              at android.os.Handler.handleCallback(Handler.java:751)
                                                                              at android.os.Handler.dispatchMessage(Handler.java:95)
                                                                              at android.os.Looper.loop(Looper.java:154)
                                                                              at android.app.ActivityThread.main(ActivityThread.java:6119)
                                                                              at java.lang.reflect.Method.invoke(Native Method)
                                                                              at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:886)
                                                                              at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:776)
                                                                           Caused by: android.os.TransactionTooLargeException: data parcel size 881128 bytes
                                                                              at android.os.BinderProxy.transactNative(Native Method)
                                                                              at android.os.BinderProxy.transact(Binder.java:615)
                                                                              at android.app.ActivityManagerProxy.activityStopped(ActivityManagerNative.java:3636)
                                                                              at android.app.ActivityThread$StopInfo.run(ActivityThread.java:3773)
                                                                              at android.os.Handler.handleCallback(Handler.java:751) 
                                                                              at android.os.Handler.dispatchMessage(Handler.java:95) 
                                                                              at android.os.Looper.loop(Looper.java:154) 
                                                                              at android.app.ActivityThread.main(ActivityThread.java:6119) 
                                                                              at java.lang.reflect.Method.invoke(Native Method) 
                                                                              at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:886) 
                                                                              at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:776) 
03-17 21:15:25.887 4492-29906/? E/ctxmgr: [ProducerActiveIntervalImpl]closeActiveInterval: Error: ongoing, trying to close
03-17 21:15:26.774 28679-28747/? E/chromium: [ERROR:ssl_client_socket_impl.cc(1101)] handshake failed; returned -1, SSL error code 1, net_error -107
03-17 21:15:26.778 28679-28747/? E/chromium: [ERROR:ssl_client_socket_impl.cc(1101)] handshake failed; returned -1, SSL error code 1, net_error -107
03-17 21:15:26.846 397-397/? E/lowmemorykiller: Error writing /proc/29174/oom_score_adj; errno=22
</details>

--- a/app/src/main/java/io/github/akameco/qiitax/MainActivity.java
+++ b/app/src/main/java/io/github/akameco/qiitax/MainActivity.java
@@ -59,8 +59,16 @@ public class MainActivity extends AppCompatActivity {
 
 	@Override
 	protected void onSaveInstanceState(Bundle outState) {
-		outState.putParcelableArrayList(STATE_ITEM_LIST, itemList);
+		storeSaveInstanceState(outState);
 		super.onSaveInstanceState(outState);
+	}
+
+	private void storeSaveInstanceState(Bundle outState) {
+		for (Item item : itemList) {
+			item.body = null;
+			item.rendered_body = null;
+		}
+		outState.putParcelableArrayList(STATE_ITEM_LIST, itemList);
 	}
 
 	private void initView() {

